templates/base.jinja2

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
```

Comment:
huh.  learned something today.  I went to check on this syntax and I'm behind the times.  The `/>` at the end is required in XML-flavored XHTML, but is supposedly not allowed in HTML, though in HTML5 it is allowed, but generally ignored.  This means that your pre-change syntax was actually correct, and I was wrong in correcting it.